site/docs/concepts/README.md

```diff
@@ -1,255 +1,232 @@
-possibly replacing an older build under which they had been running.
+* **Derivations**: You apply a transformation to data in a collection, resulting in a new collection.
 
-### Specifications
+All of these entities are described by a **catalog**.
 
-A catalog build begins from a set of **catalog specifications**
-which define the behavior of your catalog:
-the entities it contains, like captures, collections, and materializations,
-and their specific behaviors and configuration.
+### Catalogs
 
-You define catalog specifications using either the Flow web application,
-or by directly creating and editing YAML or JSON files which are typically managed
-in a Git repository using familiar developer workflows (often called "GitOps").
+A **catalog** comprises all the components that describe how your data flows function and behave: captures, materializations, derivations, collections, schemas, tests, and more.
+In other words, it states where data comes from and goes to, whether and how it will be transformed, and how it will be stored.
 
-These files use the extension `*.flow.yaml` or simply `flow.yaml` by convention.
-As a practical benefit, using this extension activates Flow's VS Code integration and auto-complete.
-Flow integrates with VS Code for development environment support, like auto-complete,
-tooltips, and inline documentation.
+When you configure your data flow in the web app or on the command line, you're creating a **catalog** to describe all of what you've configured.
+Catalogs are later built and deployed, creating your live data flows.
 
-Depending on your catalog, you may also have TypeScript modules,
-JSON schemas, or test fixtures which are also managed in your Git repository.
+A catalog can be represented as a graph of your data flow. At minimum, a catalog must have one capture, one collection, and one materialization.
 
-Whether you use the web app or Git-managed specifications is up to you,
-and teams can switch back and forth depending on what's more familiar.
+import Mermaid from '@theme/Mermaid';
 
-:::note
-The Flow web application is currently in [private beta](https://go.estuary.dev/sign-up).
-:::
+<Mermaid chart={`
+	graph LR;
+		Capture-->Collection;
+        Collection-->Materialization;
+`}/>
 
-***
+Catalogs are written in the form of YAML specification files — either manually, by you, or generated by the Flow web app.
 
-## Collections
+#### More catalog terms
 
-**Collections** are the fundamental representation for datasets within Flow, akin to a database table.
-More technically, they're a collection of documents having a common **key** and [schema](#schemas).
+* **Build**: When any component of a catalog is created or updated, Flow creates a new catalog **build**. Every build is assigned a unique identifier called a **build ID**. When you publish an updated catalog, the previous build of that catalog may be replaced.
```

Comment:
I'm wondering if we should include builds here. What's here is technically correct, but it might be that we could consider it an implementation detail and not a concept that is relevant to users.

Reply:
I had thought we talked about builds in the UI but it looks the world only ever appears in a small amount of messaging about logs 👍 Will remove